channeldb/invoices.go

```diff
@@ -174,23 +206,10 @@ type Invoice struct {
 	// or any other message which fits within the size constraints.
 	Memo []byte
 
-	// Receipt is an optional field dedicated for storing a
-	// cryptographically binding receipt of payment.
-	//
-	// TODO(roasbeef): document scheme.
-	Receipt []byte
-
 	// PaymentRequest is an optional field where a payment request created
 	// for this invoice can be stored.
 	PaymentRequest []byte
 
-	// FinalCltvDelta is the minimum required number of blocks before htlc
-	// expiry when the invoice is accepted.
-	FinalCltvDelta int32
-
-	// Expiry defines how long after creation this invoice should expire.
-	Expiry time.Duration
```

Comment:
Don't we want to maintain these fields, as otherwise, the invoice needs to be decoded just to access them.

Reply:
we still have the fields, they've just been moved